primary.c

```diff
@@ -2,29 +2,45 @@
 #include<math.h>
 #include<stdbool.h>
 #include<string.h>
-#include <stdlib.h>
-
-bool isPrime(int number){
-	for (int i = 2; i <= sqrt(number); i++)
-	{
-		if (number % i == 0)
-			return false;
-	}
-	return true;
-}
+#include<stdlib.h>
+#include<sys/types.h>
+#include<unistd.h>
 
 int main(int argc, char *argv[]){
-	if(argc == 1){
-		printf("You don't type any number, Please type numberes throw argv");
-		return 0;
-	}
-	int num;
-	int i = 0;
-	for(i = 1; i < argc; i++){
-		num = atoi(argv[i]);
-		if(isPrime(num)){
-			printf("%d ", num);
-		}
-	}
-	return 0;
+        int pipe_fd[2];
+        pid_t p;
+
+        if (pipe(pipe_fd) == -1){
+                fprintf(stderr, "Pipe Failed");
+                return 1;
+        }
+
+        p = fork();
+        if (p < 0){
+                        fprintf(stderr, "Fork Failed");
+                        return 1;
+        }
+        else if(p == 0){
```

Comment:
Use a `switch-case`

Reply:
`switch-case` will give error because `p` isn't constant value. I found the answer here: https://bit.ly/386EOmO.

Reply:
I have no idea what they are talking about.
I have used a switch case in my code and it worked so do it.

Reply:
Updated already